mitosheet/mitosheet/code_chunks/export_to_file_code_chunk.py

```diff
@@ -15,36 +15,35 @@
 from mitosheet.utils import get_conditional_formats_objects_to_export_to_excel
 
 # This is a helper function that generates the code for formatting the excel sheet
-def get_format_code(state: State) -> list:
-    code = []
+def get_format_code(state: State, sheet_index: int) -> list:
```

Comment:
What if it took sheet_indexes: List[int]? Then the call-site could be get_format_code(post_state, list(self.sheet_index_to_export_location.keys()) -- and the code would be pretty much the same as before. Probably much simpler than the complex comprehension and filter statement below.